### DISCUSSION TITLE
Kesulitan dalam code read csv

### DISCUSSION THREAD

**Discussion by mosesnugraha** [teamalgoritma/community#514] [Q&A] [answered]
selamat sore, setelah saya melakukan install R markdown dan Judah install csv yang mau Diode untuk data visualization, namun code until read csvnya error. Kalo boleh tau ini salad dimana sehingga tidal bias di read file csvnya. Apakah ada yang

**Comment by kevbow** [comment]
Dear @mosesnugraha ,

Apakah bisa ditampilkan dalam bentuk foto masalah tersebut? Silakan lampirkan kode yang digunakan maupun tempat file disimpan.

**Comment by kevbow** [comment, marked as answer]
Halo @mosesnugraha, 
Untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini karena belum ada respon setelah 7 hari respon terakhir. Semoga dapat dipahami.
